langchain4j-elasticsearch/src/main/java/dev/langchain4j/store/embedding/elasticsearch/ElasticsearchEmbeddingStore.java

```diff
@@ -259,6 +259,13 @@ public List<EmbeddingMatch<TextSegment>> findRelevant(Embedding referenceEmbeddi
         }
     }
 
+    @Override
+    public List<EmbeddingMatch<TextSegment>> findRelevant(Object memoryId, String originText, Embedding referenceEmbedding, int maxResults, double minScore) {
+        return findRelevant(referenceEmbedding, maxResults, minScore);
```

Comment:
again, this can be a default method in `EmbeddingStore`

Reply:
yes make it default